dev/envvars.sh

```diff
@@ -1,5 +1,18 @@
 #!/bin/bash
 
+case "$(uname -s | tr '[:upper:]' '[:lower:]')" in
+  linux*)
+    if [ "$(ldd /bin/ls | grep -m1 'musl')" ]; then
+      CLRDIRNAME="linux-musl"
```

Comment:
Small naming suggestion to make clear where the values come from
```suggestion
      DOTNET_RUNTIME_ID="linux-musl"
```